electrum/lnpeer.py

```diff
@@ -1425,3 +1426,53 @@ def verify_signature(tx, sig):
         # broadcast
         await self.network.try_broadcasting(closing_tx, 'closing')
         return closing_tx.txid()
+
+    @log_exceptions
```

Comment:
this decorator is unnecessary I think -- the taskgroup should already take care of it